serial_communication/web/routes/serial_handler.py

```diff
@@ -207,7 +207,7 @@ def read_serial_data(data):
 def process_untrained_message(temp_str, new_message_number):
     """Untrained messages will be executed and deleted from stack"""
     try:
-        if "restart op" in temp_str:
+        if "restart op" in temp_str or "restart" in temp_str:
```

Comment:
Make sure you don't conflict with TTGO built in commands and please make sure you also delete this message from ttgo memory, because for some reason if you are unable to delete this message then your module might stick in loop in which it just keep rebooting on this message (by terminator function used in TTGO code) if you don't delete this message.

Reply:
I do reboot is the command to reboot ttgo